custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html

```diff
@@ -80,6 +80,6 @@
         </div>
       </div>
     </div>
-    <mobile-filters selected-locations="$ctrl.selectedLocations" ng-show="$ctrl.filtersOpen">></mobile-filters>
   </div>
+  <mobile-filters selected-locations="$ctrl.selectedLocations" ng-show="$ctrl.filtersOpen">></mobile-filters>
```

Comment:
these were getting hidden behind the `if tab == chart` logic on the outer div.

Reply:
sorry you're right, this comment was incorrect. they were being hidden by `ng-show="!$ctrl.filtersOpen"` in the parent div, not the `tab === chart`